src/Tribe/Plugins.php

```diff
@@ -131,5 +131,37 @@ public function get_list() {
 			return apply_filters( 'tribe_plugins_get_list', $this->tribe_plugins );
 		}
 
+		/**
+		 * Checks if given plugin is active. Usually a Modern Tribe plugin.
+		 *
+		 * @param string $plugin_name The name of the plugin. Each plugin defines their name upon hooking on the filter.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool True if plugin is active. False if plugin is not active.
+		 */
+		public static function is_active( $plugin_name ) {
+			if ( ! did_action( "plugins_loaded" ) ) {
+				_doing_it_wrong(
+					__METHOD__,
+					__( 'Using this function before "plugins_loaded" action has fired can return unreliable results.', 'tribe-common' ),
+					'TBD'
+				);
+			}
+
+			/**
+			 * The array that each Tribe plugin overrides to
+			 * set itself as active when this function is called.
+			 *
+			 * @example [ 'the-events-calendar' => true, 'event-tickets' => true ]
+			 *
+			 * @since   TBD
+			 *
+			 * @return array Plugin slugs as keys and bool as value for whether it's active or not.
+			 */
+			$plugins = apply_filters( 'tribe_active_plugins', [] );
+
+			return isset( $plugins[ $plugin_name ] ) && $plugins[ $plugin_name ];
```

Comment:
```suggestion
			return !empty( $plugins[ $plugin_name ] );
```

Reply:
To make this even more robust we might even use `tribe_is_truthy` here:
```suggestion
			return isset( $plugins[ $plugin_name ] ) && tribe_is_truthy( $plugins[ $plugin_name ] );
```

This would allow filters to mark a plugin as active setting the value to `yes` (and your code already supported this), but it would support setting a plugin as inactive setting its value to `no`.